### PR TITLE
Fix: table scroll

### PR DESCRIPTION
**Pull request by baomastr**

### CODE REVIEW

File: src/components/v2/Table/styles.ts [hunk 1]
```diff
@@ -18,9 +18,8 @@ export const useStyles = () => {
       .MuiTableCell-root {
         border-width: 0;
         font-weight: ${theme.typography.body1.fontWeight};
-        > * {
-          float: right;
-        }
+        text-align: right;
+        font-size: 14px;
```
Comment (therealemjy) [inline src/components/v2/Table/styles.ts]: Let's use the theme

File: src/components/v2/Table/styles.ts [hunk 2]
```diff
@@ -29,7 +28,12 @@ export const useStyles = () => {
 
       .MuiTableCell-root:first-child {
         padding-left: ${theme.spacing(6)};
-        float: left;
+        text-align: left;
+        max-width: 110px;
```
Comment (therealemjy) [inline src/components/v2/Table/styles.ts]: Two things:
- we've agreed to use `theme.spacing` to express values like this
- we shouldn't have hard-coded values like this in my opinion. At the moment I'm guessing you're updating this component so it displays well on the dashboard, however that component will be reused on other screens that won't use the same columns. We should think of a way to make this component reusable

File: src/components/v2/Table/styles.ts [hunk 3]
```diff
@@ -29,7 +28,12 @@ export const useStyles = () => {
...
+      }
+
+      .MuiTableCell-root:nth-child(2) {
+        max-width: 100px;
```
Comment (therealemjy) [inline src/components/v2/Table/styles.ts]: Same thing about as previous comment

File: src/components/v2/Table/styles.ts [hunk 4]
```diff
@@ -82,5 +86,13 @@ export const useStyles = () => {
         color: ${active ? theme.palette.interactive.success : theme.palette.text.primary};
       }
     `,
+    cellWrapper: css`
+      max-width: 115px;
```
Comment (therealemjy) [inline src/components/v2/Table/styles.ts]: we've agreed to use theme.spacing to express values like this

File: src/components/v2/Table/styles.ts (hunk 1 above)
Comment (baomastr) [reply src/components/v2/Table/styles.ts]: done

File: src/components/v2/Table/styles.ts (hunk 2 above)
Comment (baomastr) [reply src/components/v2/Table/styles.ts]: added gridTemplateColumns prop that handles columns width.

File: src/components/v2/Table/styles.ts (hunk 3 above)
Comment (baomastr) [reply src/components/v2/Table/styles.ts]: removed

File: src/components/v2/Table/styles.ts (hunk 4 above)
Comment (baomastr) [reply src/components/v2/Table/styles.ts]: removed

File: src/components/v2/Table/index.tsx [hunk 5]
```diff
@@ -28,6 +28,7 @@ export interface ITableProps {
   };
   rowOnClick?: (row: ITableRowProps[]) => void;
   className?: string;
+  gridTemplateColumns?: string;
```
Comment (baomastr) [inline src/components/v2/Table/index.tsx:31]: this prop now provides handling of columns width by parent components

**Comment by therealemjy** [comment]
Looks good to me. Are you planning on fixing the tables on the dashboard in a next PR then?

**Comment by baomastr** [comment]
> Are you planning on fixing the tables on the dashboard in a next PR then?

sure. it will be the next step in https://github.com/VenusProtocol/venus-protocol-interface/pull/361

**Review by therealemjy — APPROVED** [review]

**Pull request merged**